frontend/src/Settings/index.jsx

```diff
@@ -1,16 +1,164 @@
-import React from 'react';
-import { View, Text } from 'react-native';
-import styles from '../common/styles';
-import NotificationTestExample from '../Notification/NotificationTestExample';
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { useNavigation } from 'react-navigation-hooks';
+import {
+  View, Text, Alert, StyleSheet, ImageBackground,
+} from 'react-native';
+import validation from '../common/helper';
+import Colors from '../common/colors';
+
+import background from '../../assets/login/background.jpg';
+
+import InputField from './InputField';
```

Comment:
a generic component has been merged with the ForgottenPassword branch, it's the `common/InputField`, please use that